zap-desktop-getting-started.md

```diff
@@ -1,92 +1,67 @@
+    <img src='https://imgur.com/FGvdkmT.png' alt='screenshot' />
+  </p>
 
-But for this tutorial we are going to use "Default", which will use the LND-0.4.2-beta on testnet that's packaged with the app.
+  This is part of our all new onboarding, allowing users to create and manage multiple local, remote, and BTCPay server wallets.
 
-## Step 2: Node nickname
-Next you can set your node's nickname. However it is important to remember that these nicknames are NOT unique and one should _only_ verify nodes by their public key. This nickname will often times be shown in explorers and other Lightning UIs as it can sometimes improve the user experience. I will set mine to "jimmymow" which is my online handle.
+  For this tutorial you are going to select "Create New Wallet", which will use the LND-0.5.1-beta on testnet that's packaged within Zap.
 
-<p align='center'>
-  <img src='https://i.imgur.com/A4bRI2l.png' alt='screenshot' />
-</p>
+  ## Step 2: Save your wallet seed
+  Now you're given a 24 word seed that can be used to recover your wallet. Write this down and secure it somewhere safe. If anything ever happens to your desktop or wallet you can always recover on-chain funds with this seed.
 
-## Step 3: Autopilot
-Now we can decide if we would like [autopilot](https://github.com/lightningnetwork/lnd/tree/master/autopilot) on. Autopilot is an automatic channel manager that will use 60% of your wallet balance and open 5 channels max for you on the network. We strongly recommend using autopilot as it will make the onbaording easier and overall UX better, but of course it is not required. I will enable it for this tutorial but feel free to disable it if you'd like.
+  <p align='center'>
+    <img src='https://imgur.com/2zSSHWL.png' alt='screenshot' />
+  </p>
 
-<p align='center'>
-  <img src='https://i.imgur.com/qRrjRKs.png' alt='screenshot' />
-</p>
+  ## Step 3: Verify your seed
+  Next, Zap will ask you to verify your seed by prompting you to enter three randomly selected words from your seed. Once entered correctly you'll see the inputs turn green and you can proceed. If you've entered the words incorrectly the Zap UI will turn the inputs red. 
 
-## Step 4: Wallet password
-Now we will set our wallet password. This password will be used to encrypt your wallet data. Must be 8 characters in length. You'll use this password to unlock your wallet on app start up.
+  <p align='center'>
+    <img src='https://imgur.com/cRfgaL3.png' alt='screenshot' />
+  </p>
 
-<p align='center'>
-  <img src='https://i.imgur.com/Gm4SBBX.png' alt='screenshot' />
-</p>
+  ## Step 4: Wallet password
+  Now you will set your wallet password. This password will be used to encrypt your wallet data. The password must be eight characters in length. You'll use this password to unlock your wallet on start up. If you lose this password the only way to gain access to your wallet is by way of recovery via your 24 word seed. Please back up this password and keep it safe!
```

Comment:
must be *at least* chars (not exactly 8)

Reply:
👌